app/services/location_maps_generator_service.rb

```diff
@@ -20,22 +20,39 @@ def generate
     end
 
     lib_display = {}
-    longer_display = {}
+    locations_display = {}
     holding_library = {}
     locations.each do |holding|
       holding_code = holding['code']
       lib_label = holding.library['label']
-      holding_label = holding['label'] == '' ? lib_label : lib_label + ' - ' + holding['label']
+      holding_label = holding['label'] == '' ? lib_label : holding['label']
```

Comment:
Is there a test case where `holding['label']` is empty? Curious why we account for this.

Reply:
I will look but I don't think I saw one. If there is no empty holding['label'] I'll remove the check.